target/linux/ath79/nand/base-files/etc/board.d/01_leds

```diff
@@ -8,6 +8,13 @@ board=$(board_name)
 boardname="${board##*,}"
 
 case "$board" in
+domywifi,dw33d)
+	ucidef_set_led_usbport "usb0" "USB0" "$boardname:blue:usb0" "usb1-port1"
+	ucidef_set_led_usbport "usb1" "USB1" "$boardname:blue:usb1" "usb2-port1"
+	ucidef_set_led_netdev "internet" "INTERNET" "$boardname:blue:internet" "eth0"
+	ucidef_set_led_wlan "wlan2g" "WLAN-2.4G" "$boardname:blue:wlan-2g" "phy1tpt"
+	ucidef_set_led_wlan "wlan5g" "WLAN-5G" "ath10k-phy0" "phy0tpt"
```

Comment:
What happened to the internet LED?

Reply:
Despite, I wonder whether it would be possible to set up ath10k-leds similar to what is done with ath9k-leds for wdr4900-v2:
https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/qca9558_tplink_tl-wdr4900-v2.dts#L60
https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/qca9558_tplink_tl-wdr4900-v2.dts#L101

But don't hit me if it won't work.

Reply:
> Despite, I wonder whether it would be possible to set up ath10k-leds similar to what is done with ath9k-leds for wdr4900-v2:
> https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/qca9558_tplink_tl-wdr4900-v2.dts#L60
> https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/dts/qca9558_tplink_tl-wdr4900-v2.dts#L101
> 
> But don't hit me if it won't work.

Hello, it is completely feasible to define wlan5g led in dts.